miso-web/src/main/webapp/pages/editSample.jsp

```diff
@@ -183,17 +183,18 @@
       </td>
     </tr>
     <tr>
-      <td>Project ID:</td>
+      <td>Project:</td>
       <c:choose>
         <c:when test="${empty sample.project}">
           <td>
-            <div id="projectlist" class="checklist">
-              <form:checkboxes items="${accessibleProjects}" path="project" itemValue="id"
-                               itemLabel="name" onclick="Utils.ui.uncheckOthers('project', this);"/>
-            </div>
-            <div class="parsley-errors-list filled" id="projectError">
-              <div class="parsley-required"></div>
-            </div>
+            <form:select id="project" path="project" onchange="Sample.ui.projectChanged();">
```

Comment:
I thought a few lines here made more sense than a few lines of Java somewhere else to only be useful here? miso:select is nice because it normally reduces the code. Moving the code doesn't seem like an improvement